### PR TITLE
Update several examples to work with the new API

### PR DESCRIPTION
**Pull request by vivshaw**
I got several of the examples fixed up to work with the new API; let me know whether there are any problems

### CODE REVIEW

File: examples/imdb_cnn_lstm.py [hunk 1]
```diff
@@ -55,12 +55,12 @@
...
 model.add(Dropout(0.25))
-model.add(Convolution1D(filters=filters,
-                        filter_length=filter_length,
-                        border_mode='valid',
-                        activation='relu',
-                        subsample_length=1))
-model.add(MaxPooling1D(pool_length=pool_length))
+model.add(Conv1D(filters,
+                 kernel_size,
+     	         padding='valid',
```
Comment (fchollet) [inline examples/imdb_cnn_lstm.py]: PEP8 issue.

**Comment by fchollet** [comment]
Thanks! The changes all look good as far as I can tell (though from the GH PR interface I couldn't tell if they cover everything). Please run a PEP8 linter on your code, as there is at least once PEP8 issue and possibly more that I missed.

**Comment by vivshaw** [comment]
Oops, sorry about the PEP8 fail there! Should be properly fixed now.

**Comment by fchollet** [comment]
Thanks!

**Comment by MoyanZitto** [comment]
@vivshaw
Hi，where can I find the new APIs?

**Pull request merged**